components/compliance-service/ingest/ingestic/ingestic.go

```diff
@@ -580,7 +596,7 @@ func getPercentageComplete(status interface{}) (float64, bool) {
 		return 1, true
 	}
 
-	return (created + deleted + updated) / total, true
+	return (created + deleted + updated + conflicts + noops) / total, true
```

Comment:
does this make our time estimate more accurate?

Reply:
the previous logic ignored these values and freaked out when a task was completed but didn't have `created + deleted + updated = total`
We need to account for the other things because conflicts are normal and its possible that the script decides it doesn't want to change a doc, which is a noop (though none of our scripts noop)